docs/server/source/appendices/run-with-docker.md

```diff
@@ -25,7 +25,7 @@ docker run \
   --interactive \
   --rm \
   --tty \
-  --volume "$HOME/bigchaindb_docker:/data" \
+  --volume $HOME/bigchaindb_docker:/data \
```

Comment:
There were double quotes in some places and missing in others, so I just removed it everywhere to make it uniform.